master/reference/calicoctl/resources/policy.md

```diff
@@ -47,14 +50,15 @@ spec:
-| ingress    | Ordered list of ingress rules applied by policy.                                                                                                      |                 | List of [Rule](#rule) |         |
-| egress     | Ordered list of egress rules applied by this policy.                                                                                                  |                 | List of [Rule](#rule) |         |
-| doNotTrack | Indicates to apply the rules in this policy before any data plane connection tracking, and that packets allowed by these rules should not be tracked. | true, false     | boolean               | false   |
-| preDNAT    | Indicates to apply the rules in this policy before any DNAT.                                                                                          | true, false     | boolean               | false   |
+| Field      | Description                                                                                                                                           | Accepted Values | Schema                | Default  |
+|------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|-----------------------|----------|
+| order      | Indicates ordering of this policy: policies with lower order are applied earlier. No value means that the policy is applied last.                     |                 | float                 |          |
+| selector   | Selects the endpoints to which this policy applies.                                                                                                   |                 | [selector](#selector) | `all()`  |
+| ingress    | Ordered list of ingress rules applied by policy.                                                                                                      |                 | List of [Rule](#rule) |          |
+| egress     | Ordered list of egress rules applied by this policy.                                                                                                  |                 | List of [Rule](#rule) |          |
+| doNotTrack | Indicates to apply the rules in this policy before any data plane connection tracking, and that packets allowed by these rules should not be tracked. | `true, `false`  | boolean               | `false`  |
+| preDNAT    | Indicates to apply the rules in this policy before any DNAT.                                                                                          | `true, `false`  | boolean               | `false`  |
+| types      | Specifies explicitly whether the policy applies to ingress traffic, or to egress, or to both. | | List of strings, which may each be 'ingress' or 'egress'. | See below. |
```

Comment:
Ditto for this table.  Put `types` after `selector` but before `ingress`.

Reply:
done.